### DISCUSSION TITLE
Adding To SNMP notification

### DISCUSSION THREAD

**Discussion by ahozdemir** [trimble-oss/dba-dash#618] [Ideas]
Hi,

Is there an any plan to implement enterprise notification via SNMP,  like when any alert is appears then SNMP send trap through alert message,
I used a lot of enterprise stuff like quest foglight, redgate sql monitor, apexsql monitor and so on, some of complicated some of disability to do some basic stuff ( like basic template, report )

**Comment by DavidWiseman** [comment]
Hi, sorry for the slow response.  I don't have any plans to implement SNMP notifications.
